external-api/src/interfaces/number.interface.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Defines the interfaces for the response of the number generator.
+ *
+ * @interface NumberGeneratorResponse
+ * @property {number} status - The status code of the response.
+ * @property {Array} data - An array of objects containing the generated numbers.
+ * @property {number} data.id - The ID of the generated number.
+ * @property {number} data.maskedId - The masked ID of the generated number.
+ * @property {number} data.type - The type of the generated number.
+ * @property {string} data.createdBy - The creator of the generated number.
+ * @property {string} data.createdDatetime - The creation datetime of the generated number.
+ * @property {string} data.requestingSystem - The system that requested the generation of the number.
+ */
+export interface NumberGeneratorResponse {
+  readonly status: number;
+  readonly data: Array<{
+    id: number;
```

Comment:
Is this ID going to be one like '0010048579'? If so is it not going to be a string rather than a number especially for trailing zeroes?

Reply:
ID is a number 
![image](https://github.com/UK-Export-Finance/dtfs2/assets/32125108/5fa597e8-1ed0-44f0-b8b3-dd2141eb3034)